images/build/debian-iptables/buster/Dockerfile

```
@@ -18,6 +18,7 @@ FROM ${BASEIMAGE}
 
 # Install iptables and ebtables packages from buster-backports
 ARG IPTABLES_VERSION
+RUN ln -s /bin/rm /usr/sbin/rm
 RUN echo deb http://deb.debian.org/debian buster-backports main >> /etc/apt/sources.list \
     && apt-get update \
     && apt-get -t buster-backports -y --no-install-recommends install \
```

My recommendation would be to run these in the same command as each of these is a new layer.
```suggestion
RUN ln -s /bin/rm /usr/sbin/rm \
     && echo deb http://deb.debian.org/debian buster-backports main >> /etc/apt/sources.list \
    && apt-get update \
    && apt-get -t buster-backports -y --no-install-recommends install \
```

---

Good call, changed